ada_feeding_perception/ada_feeding_perception/segment_from_point.py

```diff
@@ -78,7 +78,28 @@ def __init__(self) -> None:
         self.active_goal_request_lock = threading.Lock()
         self.active_goal_request = None
 
-        # Subscribe to the image topic, to store the latest image
+        # Subscribe to the camera info topic, to get the camera intrinsics
+        self.camera_info_subscriber = self.create_subscription(
+            CameraInfo,
+            "/camera/color/camera_info",
+            self.camera_info_callback,
+            1,
+        )
+        self.camera_info = None
+        self.camera_info_lock = threading.Lock()
+
+        # Subscribe to the aligned depth image topic, to store the latest depth image
+        # NOTE: We assume this is in the same frame as the RGB image
+        self.depth_image_subscriber = self.create_subscription(
+            Image,
+            "/camera/aligned_depth_to_color/image_raw",
```

Comment:
Are all these topic names hard-coded? 

A better tactic would be to have these strings be "image", "aligned_depth", and "camera_info", and remap them in a roslaunch.

This isn't required right now, but add it as an Issue before Merging so we don't forget.

Reply:
(Unless it's fast to update)